lib/view/ui/splash_screen.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:community/utils/app_string_res.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:community/utils/color.dart';
+import 'package:community/utils/dimen.dart';
+import 'package:community/view_model/splash_controller.dart';
+
+class SplashScreen extends StatelessWidget {
+
+  final SplashController splashController = Get.put(SplashController());
+  @override
+  Widget build(BuildContext context) {
+          return Scaffold(
+      body: Center(
+        child:
+            Container(width: Dimen_200, height: Dimen_300,
+                child: Column(
+                  children: [
+                    Image.asset('assets/images/ic_splash_logo.png'),
```

Comment:
Again, this is not correct way of using image assets please check online. [it](https://maruf-hassan.medium.com/handling-flutter-images-like-a-pro-c28460b2806)